src/components/ChatBox.tsx

```diff
@@ -0,0 +1,58 @@
+import useSessionComm from "~/hooks/useSessionComm";
+
+const Chatbox = ({
+  sessionId,
+  userId,
+  userName,
+  className,
+}: {
+  sessionId: string;
+  userId: string;
+  userName: string;
+  className?: string | undefined;
+}) => {
+  const [
+    allSessionMessages,
+    sendMessage,
+    onTyping,
+    currentMessage,
+    userTypingMessage,
+  ] = useSessionComm(sessionId, userId, userName);
+
+  return (
+    <div className={className}>
+      <div className="messages-container overflow-y-auto flex flex-col h-[18.5rem]">
```

Comment:
idt we should be using fixed height for this div?

Reply:
i changed to the outer div having fixed height

Reply:
Why is fixed height needed anyways? can't we make do with min-height?